libmei/tools/cpp.py

```diff
@@ -1214,7 +1214,7 @@ class MEIBasic {{
 
 
 def create_basic_validator(configure: dict, outdir: Path):
-    basic_path = Path(configure["basic_odd"])
+    basic_path = Path(configure["config_dir"]) / Path(configure["basic_odd"])
```

Comment:
It's a style thing, but I really really dislike the division operator overrides for path segments.

Reply:
Should I use `joinpath` or what exactly is the best alternative?

Reply:
In this specific case, something like:

`Path(configure["config_dir"], configure["basic_odd"])`

would do the same thing.